### ISSUE TITLE
No audio output on air unit firmware version 0608

### ISSUE THREAD

**Issue by sigmund101** [fpv-wtf/wtfos-configurator#417]
### Device

Air Unit

### Operating System (Including version)

Windows 11

### Browser

Chrome

### Describe the issue

Air unit does not output sound to goggles after upgrading the air unit to firmware 01.00.0608.    Reverting firmware on air unit to either 01.00.0606 or 01.00.0600 restores audio output.  Was able to replicate issue on multiple drones/air units.    

Edited for clarity.

**Comment by sigmund101** [comment]
Yes I am referring to air unit firmware.     Ex:  GogglesV1 01.00.0608 + AirUnit 01.00.0606 = Audio works            Goggles 01.00.0608 + AirUnit 01.00.0608 = No Audio

**Comment by j005u** [comment]
If you use butter to flash 0606 over 0608 that should skip updating the reported FW version, thus fixing any goggles warning popup, while restoring audio functionality: https://bin.fpv.tools/butter/packages/wm150_01.00.0606_recovery.zip

0608 is technically only necessary if you're using the Runcam Link camera. Otherwise the above combination should be fine.

Will need to see if there's anything we can do to fix this issue on 0608, can't make any promises. Leave this issue open till then.
